do_like_javac/tools/wpi.py

```diff
@@ -54,7 +55,7 @@ def run(args, javac_commands, jars):
         iteration = 0
         diffResult = 1
         stubDirs = []
-        resultsDir = tempfile.mkdtemp(prefix="wpi-stubs-")
+        resultsDir = tempfile.mkdtemp(prefix="wpi-stubs-" + datetime.now().format("%Y%m%d%H%M%S)"))
```

Comment:
@kelloggm The problem is an extra close paren within the string (after "%S").

Reply:
The real problem is why CI didn't catch it. Fortunately, GitHub Actions is conveniently "experiencing issues" right now 🙃